skbio/io/format/gff3.py

```diff
@@ -0,0 +1,469 @@
+r'''
+GFF3 format (:mod:`skbio.io.format.gff3`)
+=========================================
+
+.. currentmodule:: skbio.io.format.gff3
+
+GFF3 is a standard file format for storing genomic features in a text
```

Comment:
thought it was just a comment rather than a request of change, because somehow it was and is still only showing your comments out of context without Jai's first comment. now I am able to see them all with some tricks. rewording.